frontends/web/src/components/guide/guide.tsx

```diff
@@ -61,14 +62,30 @@ interface ProvidedProps {
 
 type Props = ProvidedProps & SharedProps & TranslateProp;
 
+// guide entries are fetched from the translation files. Manual fallback to is implemented in case an
+// array element is undefined in the current language (can be null if not translated in order, for
+// example).
+function getGuideEntries(t, screen): ImplicitEntry[] {
+    const key = 'guide.' + screen;
+    // For now, fallback to the main fallback language, not cascading.
+    const entriesFallback = i18n.getFixedT(i18n.languages[i18n.languages.length - 1])(key, { defaultValue: [] });
```

Comment:
For consistency, `entriesFallback` should be renamed to `entriesFallbackLng` respectively `entriesFallbackLanguage`. Code (including variables) should speak as much as they can for themselves. 🕺